pom.xml

```diff
@@ -3,8 +3,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-      <!--JENKINS-18537-->
-    <version>1.554.1</version>
+    <version>1.580.1</version>
```

Comment:
@amuniz a tip: if you change this line to

```xml
<version>1.580.1</version> <!-- first LTS after SCMTriggerItem -->
```

then this whole comment thread becomes hidden by default. :-)